tests/unit/core/selector.js

```diff
@@ -130,14 +130,14 @@ test( "focusable - hidden styles", function() {
 	isNotFocusable( "#displayNoneAncestor-input", "input, display: none parent" );
 	isNotFocusable( "#displayNoneAncestor-span", "span with tabindex, display: none parent" );
 
-	isNotFocusable( "#visibilityHiddenAncestor-input", "input, visibility: hidden parent" );
-	isNotFocusable( "#visibilityHiddenAncestor-span", "span with tabindex, visibility: hidden parent" );
+	isFocusable( "#visibilityHiddenAncestor-input", "input, visibility: hidden parent" );
+	isFocusable( "#visibilityHiddenAncestor-span", "span with tabindex, visibility: hidden parent" );
```

Comment:
None of these elements have `visibility: visible` on them, so they're all still hidden.